cmd/generate/generate.go

```diff
@@ -198,6 +198,10 @@ type Schema struct {
 	K8sRoleBindingList           rbac.RoleBindingList
 	NetNameSpace                 networkapi.NetNamespace
 	NetNameSpaceList             networkapi.NetNamespaceList
+	K8sClusterRole               rbac.ClusterRole
+	K8sClusterRoleList           rbac.ClusterRoleList
```

Comment:
Here adding only List element is enough, like you just add `K8sClusterRoleList` and `K8sClusterRoleBindingList` will automatically add `K8sClusterRole` and `K8sClusterRoleBinding` repectively